src/App.js

```diff
@@ -1,5 +1,12 @@
 import React from 'react'
 import styled from 'styled-components'
+import monitorIcon from './icon/monitor.svg'
+import locationIcon from './icon/location.svg'
+import letterIcon from './icon/letter.svg'
+import cubeIcon from './icon/cube.svg'
+import worldIcon from './icon/world.svg'
+import githubIcon from './icon/github.svg'
+import codepenIcon from './icon/codepen.svg'
```

Comment:
In general, die Namen des icon besser nennen für ein besser verstandnis. ex. monitorIcon = companyIcon, letterIcon = emailIcon etc. cubeIcon und WorldIcon auch mit ein andere Name.